tools/devops/device-tests/templates/device-tests.yml

```diff
@@ -246,24 +246,24 @@ steps:
 - bash: |
     set -x
     set -e
-    echo 'Use xamarin storage $USE_XAMARIN_STORAGE'
+    P=$(echo $XM_STORAGE_PATH | tr -d \'\")
 
     cd $WORKING_DIR/xamarin-macios
     if [[ "$USE_XAMARIN_STORAGE" == "True" ]]; then
       if nc -z xamarin-storage 22 2>/dev/null; then
-        EC=0
-        MKSTORAGE='ssh builder@xamarin-storage "mkdir -p /volume1/storage/$XAMARIN_STORAGE_PATH"'
-        eval $MKSTORAGE || EC=$?
-        if [ $EC -eq 0 ]; then
-          echo '##vso[task.setvariable variable=XAMARIN_STORAGE_PATH;isOutput=true]$XAMARIN_STORAGE_PATH'
-          echo '##vso[task.setvariable variable=XAMARIN_STORAGE_FAILED;isOutput=true]false'
-          export TESTS_PERIODIC_COMMAND="--periodic-interval 10 --periodic-command rsync --periodic-command-arguments '-avz -e \"ssh\" $PWD/jenkins-results builder@xamarin-storage:/volume1/storage/$XAMARIN_STORAGE_PATH'"
+        ssh builder@xamarin-storage "mkdir -p /volume1/storage/$P"
+        if [[ $? = 1 ]]; then
+          echo "Remote mkdir failed"
+          echo "##vso[task.setvariable variable=XAMARIN_STORAGE_FAILED;isOutput=true]true"
+          echo "##vso[task.setvariable variable=XAMARIN_STORAGE_PATH;isOutput=true]"
```

Comment:
This clears the var even w/o the empty string at the end?

Reply:
yes